test/e2e/specs/editor/blocks/image.spec.js

```diff
@@ -312,7 +313,12 @@ test.describe( 'Image', () => {
 
 		// Assert that the image is initially unscaled and unedited.
 		const initialImageSrc = await image.getAttribute( 'src' );
-		const initialImageDataURL = await imageBlockUtils.getDataURL( image );
+		await expect
+			.poll( () => image.boundingBox() )
```

Comment:
nit: We can drop the extra wrap
```suggestion
			.poll( image.boundingBox )
```

Reply:
Nope, I tried that but it failed because of missing `this` in the `boundingBox` implementation.